angular/src/directives/show-more.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, ElementRef, Input, Renderer2, } from '@angular/core';
```

Comment:
I'm not sure what this file is, it looks like you've copied the existing `A11yTitleDirective` but haven't done anything more with it. Should it be removed?